frontend/src/styles/components/SelectionTool.scss

```
@@ -126,102 +173,71 @@
+    width: 100%;
+    box-sizing: border-box;
   }
 
   &__helper {
-    margin-top: 0.5rem;
-    font-size: 0.9rem;
+    padding: 1rem 0;
+    font-size: 1rem;
     color: rgba(15, 23, 42, 0.6);
+    min-width: 0;
+    word-wrap: break-word;
```

Stylelint (17.3.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)